content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,95 @@
+<!-- overview -->
+
+This document describes the concept of a VolumeAttributesClass in Kubernetes. Familiarity with [storage classes](https://kubernetes.io/docs/concepts/storage/storage-classes/), [volumes](https://kubernetes.io/docs/concepts/storage/volumes/) and [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) is suggested.
+
+<!-- body -->
+
+## Introduction
+
+A VolumeAttributesClass provides a way for administrators to describe the mutable "classes" of storage they offer. Different classes might map to quality-of-service levels. Kubernetes itself is unopinionated about what classes represent. 
+
+
+## The VolumeAttributesClass Resource
+
+Each VolumeAttributesClass contains the `driverName` and `parameters`, which are used when a PersistentVolume belonging to the class needs to be dynamically provisioned or modified.
+
+The name of a VolumeAttributesClass object is significant, and is how users can request a particular class. Administrators set the name and other parameters of a class when first creating VolumeAttributesClass objects. While the name of a VolumeAttributesClass object in a PersistentVolumeClaim is mutable, the parameters in an existing class are immutable.
+
+
+```yaml
+apiVersion: storage.k8s.io/v1alpha1
+kind: VolumeAttributesClass
+metadata:
+  name: silver
+driverName: pd.csi.storage.gke.io
+parameters:
+  provisioned-iops: "3000"
+  provisioned-throughput: "50" 
+```
+
+
+### Provisioner
+
+Each VolumeAttributes has a provisioner that determines what volume plugin is used for provisioning PVs. This field must be specified. 
```

Comment:
Where does this `VolumeAttributes` come from?
Is it always a plural?

Reply:
It is what we decided in the KEP design. Because these are volume attributes like iops, throughput, type etc. but they are all different depending on different vendors

Reply:
VolumeAttributes -> VolumeAttributesClass?

Reply:
The "provisioner" field you referred here is the "driverName" field?  Can you clarify?

Reply:
okay let me add driverName